src/client.ts

```diff
@@ -0,0 +1,620 @@
+import axios, { AxiosError, AxiosInstance } from 'axios'
+import {
+  AddedNodeInfo,
```

Comment:
`import * as BitcoinRpc from '.'` ?

Reply:
would need to move your type alias into another file however to avoid cycle

Reply:
Don't like that so much then I need to prefix everything IIRC?

Reply:
yes, it depends if you prefer a long list of imports or a small prefix everywhere like `BR.XXXX`.
as you want

Reply:
I prefer the long list :)